plugins/CoreHome/angularjs/widget/widget.directive.js

```diff
@@ -109,7 +109,7 @@
                             var title = $(this).attr('title');
                             return piwikHelper.escape(title.replace(/\n/g, '<br />'));
                         },
-                        show: {delay: 700, duration: 200},
+                        show: {delay: 100, duration: 200},
```

Comment:
is that needed? We set it to 700 on purpose to simulate browser behaviour a bit more. We don't want them to show up so quick as it can be quite annoying.

Reply:
Note: the tooltip doesn't come up immediately over hover, might be required to try it locally and see if it's ok or not...

Reply:
Did we get those complaints before? Still think it would be good to try and see if it's an issue. Don't need an answer immediately or anything, though.

Reply:
I don't know how this setting may relate to other widget's tooltips. The behaviour of the action tooltips up to the 3.7.0 version made it possible to browse the titles of the pages seen by the user at a glance. With a 700ms delay this becomes very frustrating. As an alternative I found myself opening the "visitor profile" but most of the time I rather prefer the previous tooltip behaviour.

Reply:
@mzcarlos this is the global tooltip class. So it will have an effect on all tooltips that are not created somewhere else.

Reply:
It is somehow possible to change it only for real time widget? There it should make sense to not use a delay basically

Reply:
@tsteur updated